matrix/lambdas/daemons/worker.py

```diff
@@ -79,13 +85,13 @@ def _parse_bundles_to_dataframes(self, bundle_uuids: typing.List[str]):
                     exp_df, qc_df = future.result()
                 except MatrixException as e:
                     self.request_tracker.log_error(e.title)
-                    raise
+                    return None, None
```

Comment:
Since we're not re-raising the exception here, we should log the full exception `e` in this case